### DISCUSSION TITLE
Can I do this without Weblate?

### DISCUSSION THREAD

**Discussion by mybearworld** [meower-media-co/i18n#35] [Polls]
Can I just create a `.po` file for German without Weblate?

**Comment by theycallhermax** [comment]
yes, just:

- figure out what project you are going to work on, they are separated into separate folders in the repo's root directory, `Meower-Svelte` and `Meower-Website`
- fork the repo
- copy and paste the `Meower-Svelte.pot` or `Meower-Website.pot` file into `Meower-Svelte/po/de.po`, or `Meower-Website/po/de.po`, depending on your decision
- add the translations to the `Meower-Svelte/po/de.po` or `Meower-Website/po/de.po` file, using `msgstr` as what to translate and `msgstr` as inputting your translation
- once you are finished, make a pr and await review

**Reply by mybearworld** [reply]
Alright, thanks!

**Reply by mybearworld** [reply]
Done: #36

**Comment by theycallhermax** [comment]
marking as resolved
